vllm/forward_context.py

```diff
@@ -41,7 +41,8 @@ class ForwardContext:
     attention layer to its attention metadata
     set dynamically for each forward pass
     """
-    attn_metadata: Union["AttentionMetadata", dict[str, "AttentionMetadata"]]
+    attn_metadata: Optional[Union["AttentionMetadata",
```

Comment:
(It looks like vllm/attention is not ready for follow_imports: "skip" -> "silent" yet, but this is enough to surface the issue in pyright/pylance)